modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-no-slip-action.i

```diff
@@ -17,11 +17,9 @@ rho=1.1
 [Modules]
   [NavierStokesFV]
     compressibility = 'incompressible'
-    porous_medium_treatment = false
-    add_energy_equation = false
 
-    density = 'rho'
-    dynamic_viscosity = 'mu'
+    density = ${rho}
+    dynamic_viscosity = ${mu}
 
     initial_velocity = '1 1 0'
```

Comment:
```suggestion
    initial_velocity = '1 0 0'
```

Reply:
I don't think I will change this because this is a test. This is a steady-state simulation so I don't think the audience will get confused if we have a not very realistic IC.